File: app/src/main/java/im/point/dotty/network/RawComment.java
package im.point.dotty.network;

import com.google.gson.annotations.SerializedName;

public class RawComment {
    String created;
    String text;
    Author author;
    @SerializedName("post id") String postId;
    @SerializedName("to comment id") int toCommentId;
    @SerializedName("is rec") boolean isRecommendation;
    int id;

    public String getCreated() {
        return created;
    }

    public String getText() {
        return text;
    }

    public Author getAuthor() {
        return author;
    }

    public String getPostId() {
        return postId;
    }

    public int getToCommentId() {
        return toCommentId;
    }

    public boolean isRecommendation() {
        return isRecommendation;
    }

    public int getId() {
        return id;
    }
}

File: app/src/main/java/im/point/dotty/network/RawComment.kt
package im.point.dotty.network

import com.google.gson.annotations.SerializedName

data class RawComment(var created: String? = null,
                      var text: String? = null,
                      var author: Author? = null,
                      var id : Int? = null,
                      @SerializedName("post_id")
                      var postId: String? = null,
                      @SerializedName("to_comment_id")
                      var toCommentId : Int? = null,
                      @SerializedName("is_rec")
                      var isRecommendation : Boolean? = null)